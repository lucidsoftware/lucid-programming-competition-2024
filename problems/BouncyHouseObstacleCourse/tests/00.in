3
A
B
C
3
A B 10
B C 7
A C 19
