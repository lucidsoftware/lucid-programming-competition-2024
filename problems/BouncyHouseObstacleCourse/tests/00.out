18
A
B
C
